solutions/security/detect-and-alert/monitor-rule-executions.md

```diff
@@ -97,18 +122,12 @@ Use these controls to filter what’s included in the logs table:
    stack: preview 9.0, ga 9.1
 ```
 
-Gaps in rule executions are periods of time where a rule didn’t run. They can be caused by various disruptions, including system updates, rule failures, or simply turning off a rule. Addressing gaps is essential for maintaining consistent coverage and avoiding missed alerts.
+Use the information in the Gaps table to assess the scope and severity of rule execution gaps. To control what's shown in the table, you can filter the table by gap status, select a time range for viewing gap data, and sort multiple columns. {applies_to}`stack: ga 9.3` Fill all gaps for the rule by clicking **Fill all gaps**.
```

Comment:
```suggestion
Use the information in the Gaps table to assess the scope and severity of rule execution gaps. To control what's shown in the table, you can filter the table by gap status, select a time range for viewing gap data, and sort multiple columns. 

{applies_to}`stack: ga 9.3` Fill all gaps for the rule by clicking **Fill all gaps**.
```

Brandon shared a best practice with me recently which is to not have applies to labels be inline. I think this makes it a bit more clear what the applies to... applies to :trollface:

Reply:
There's a :trollface: emoji?!